### PR TITLE
Completed API endpoints

### PR DESCRIPTION
**Pull request by mellava**
- Finished API endpoints 
- Create tables

### CODE REVIEW

File: api/routes/api/bookmarks.route.js
Comment (rul008) [inline api/routes/api/bookmarks.route.js:1]: we need get bookmark list

**Comment by rul008** [comment]
photo [],
zipcode INT,
hours, TEXT[]

File: api/db/schema.sql [hunk 1]
```diff
@@ -1,61 +1,79 @@
...
   password_hash TEXT,
   latitude TEXT,
-  longitue TEXT,
-  created_at TIMESTAMP,
-  updated_at TIMESTAMP
+  longitude TEXT,
+  created_at TIMESTAMP DEFAULT now()
 );
 
 CREATE TABLE restaurants (
```
Comment (fmirando) [inline api/db/schema.sql:14]: Going to need a column for categories (breakfast lunch and dinner)

File: api/db/schema.sql [hunk 2]
```diff
@@ -1,61 +1,79 @@
...
   user_id INT REFERENCES users(id),
-  created_at DATE
+  restaurant_id INT REFERENCES restaurants(id),
+  created_at TIMESTAMP DEFAULT now()
+);
+
+CREATE TABLE cuisines (
+  id SERIAL PRIMARY KEY NOT NULL,
+  name VARCHAR(50),
+  about TEXT,
```
Comment (fmirando) [inline api/db/schema.sql]: Not sure if the 'about' attribute is for but we can maybe split this up into 3 sub attributes for cultural insights. Going to need 3 more columns for the following in the cuisines table: 
-History VARCHAR(300)
-Customs VARCHAR(300)
-Dishes TEXT[] (an array of strings eg ['orange chicken', 'dim sum', 'kung pao chicken'], etc)

File: api/routes/api/bookmarks.route.js
Comment (mellava) [reply api/routes/api/bookmarks.route.js:1]: Bookmarks GET has been re-added. But bookmarks are part of the user response.

File: api/db/schema.sql (hunk 1 above)
Comment (mellava) [reply api/db/schema.sql:14]: I'll handle this in a separate PR.

File: api/db/schema.sql (hunk 2 above)
Comment (mellava) [reply api/db/schema.sql]: History and customs have been added. Dishes are the recipes.

**Comment by mellava** [comment]
> photo [], zipcode INT, hours, TEXT[]

Photos array should already exist. Zip added. Hours added.

File: api/controllers/restaurantController.js [hunk 3]
```diff
@@ -1,22 +1,84 @@
 const mockRestaurants = require('../../__mocks__/mockRestaurants');
+const {Restaurant} = require('../models');
 
 const createRestaurant = async (req, res) => {
-  res.sendStatus(200);
+  const {
+    name,
+    description,
```
Comment (fmirando) [inline api/controllers/restaurantController.js:7]: Going to need a property for category that possibly holds **breakfast**, **brunch** **lunch**, **dinner** or **dessert** unless description would already have those keywords.

File: api/controllers/userController.js [hunk 4]
```diff
@@ -1,18 +1,54 @@
 const mockUser = require('../../__mocks__/mockUser');
+const {User} = require('../models');
 
 const createUser = async (req, res) => {
```
Comment (fmirando) [inline api/controllers/userController.js:4]: Is this already integrated with Firebase Auth? Please connect with Andrew on this if you haven't already

File: api/controllers/restaurantController.js (hunk 3 above)
Comment (fmirando) [reply api/controllers/restaurantController.js:7]: Nevermind I see your reply saying that this will be in a future PR

**Review by fmirando — APPROVED** [review]
A lot of good work here, thanks Melissa

**Pull request merged**